flask/app.yaml.template

```diff
@@ -1,3 +1,10 @@
-service: <SERVICE> 
+service: <SERVICE>
 runtime: python312
 entrypoint: gunicorn -b :$PORT -w 2 --timeout 60 src.main:app
+
+env_variables:
+  REDISHOST: <REDISHOST>
+  REDISPORT: <REDISPORT>
+
+vpc_access_connector:
+  name: <VPC_ACCESS_CONNECTOR>
```

Comment:
For now please hardcode these values, I will un-hardcode later once I fix/refactor `deploy.sh`
but yeah, this is how it's going to look eventually 🙂

Reply:
roger that 🫡